packages/block-library/src/group/style.scss

```diff
@@ -2,3 +2,8 @@
 	// This block has customizable padding, border-box makes that more predictable.
 	box-sizing: border-box;
 }
+
+// We need this so groups with negative margins overlap as expected
```

Comment:
Not a blocker, but I tend to always end comments with a period. Nitpicky I know!

Reply:
I'm surprised this is not being caught by the linter then! I think the JS one does check for that